config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/ingredients", to: "ingredients#index"
+  get "/recipes/:id", to: "recipes#show"
+  patch "/recipes/:recipe_id", to: "recipes#update"
```

Comment:
It looks like you were going to set up a patch to add an ingredient to a recipe if I'm not mistaken. 

Another way to do this that is more RESTful is to do a POST to a new Joins controller rather than a patch to the recipes controller. If, in the future, you want to update the name, complexity, or genre of a recipe, you'll want that `patch` update for those main attributes. But when you want to build the relationship between an ingredient and a recipe, consider making a `recipe_ingredients_controller` and routing that form to a `create` action instead. In this case you're not updating the recipe by itself, but rather `creating` a joins table record in the recipe_ingredients table.